legacy/src/main/java/org/opensearch/sql/legacy/executor/format/DateFieldFormatter.java

```diff
@@ -105,7 +105,7 @@ private List<String> getFormatsForColumn(String columnName) {
   private Set<String> getDateColumns(List<Schema.Column> columns) {
     return columns.stream()
         .filter(column -> column.getType().equals(Schema.Type.DATE.nameLowerCase()))
-        .map(Schema.Column::getName)
+        .map(Schema.Column::getIdentifier)
```

Comment:
Is this related to the current issue?

Reply:
Good catching! This change should be reverted. What we need to change is https://github.com/opensearch-project/sql/pull/2757/files#diff-943465139286a5010c19cab380f141e3c9bc26fb02652bedbc6a7bbec5c98cc3R46